### PR TITLE
refresh lightbox

### PR DESCRIPTION
**Pull request by spasovski**
Didn't see a bug number for this.

### CODE REVIEW

File: src/media/css/lightbox.styl [hunk 1]
```diff
@@ -1,7 +1,9 @@
 @import 'lib';
 
+$close-btn-color = #f9634c;
+
 #lightbox {
-    background: rgba(0,0,0,.75);
+    background: rgba(198,198,198,.8);
```
Comment (ngokevin) [inline src/media/css/lightbox.styl]: use a palette color

File: src/media/css/lightbox.styl [hunk 2]
```diff
@@ -1,7 +1,9 @@
 @import 'lib';
 
+$close-btn-color = #f9634c;
```
Comment (ngokevin) [inline src/media/css/lightbox.styl]: palette color

**Comment by ngokevin** [comment]
Do we want a red close button? We had red cancel buttons for modals before we switched them to blue because Phil said red seemed destructive

**Comment by ngokevin** [comment]
Clicking on the modal background should close the modal IMO

**Comment by spasovski** [comment]
re: red vs blue - I'll take your word on it that it should be blue since you worked on the others. I didn't change the clicking on background close functionality so if that's regressed I'll fix0r...oh I even think I know why.

**Pull request closed without merging**